src/createEslinter.ts

```diff
@@ -37,9 +37,13 @@ export function createEslinter(eslintOptions: object) {
       | IterableIterator<eslinttypes.CLIEngine.LintReport>
       | eslinttypes.CLIEngine.LintReport[]
   ) {
+    const eslint: typeof eslinttypes = require('eslint');
```

Comment:
Is it necessary to re-`require` ESLint here? We require it earlier in `createEslinter`. Is that no longer used with the new approach?